float32 right_side
float32 right_middle
float32 right_center
float32 left_center
float32 left_middle
float32 left_side
float32 linear_x
float32 angular_z
